src/main/views/accessibility-statement.njk

```diff
@@ -0,0 +1,79 @@
+{% from "macros/breadcrumb.njk" import breadcrumb %}
+{% from "macros/csrf.njk" import csrfProtection %}
+{% from "macros/errorSummary.njk" import errorSummary %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+
+{% extends "template.njk" %}
+
+{% block pageTitle %}
+  Accessibility statement - IDAM User Dashboard - GOV.UK
+{% endblock %}
+
+{% block content %}
+  <h1 class="govuk-heading-xl">Accessibility statement for IDAM User Dashboard</h1>
+
+  <p class="govuk-body-m">This accessibility statement explains how accessible this website is, what to do if you have difficulty using it, and how to report accessibility problems with the website.</p>
+  <p class="govuk-body-m">This website is part of the wider GOV.UK website. There is a separate <a class="govuk-link" href="https://www.gov.uk/help/accessibility-statement">accessibility statement</a> for the main GOV.UK website.</p>
+  <p class="govuk-body-m">This page only contains information about the IDAM User Dashboard.</p>
+  <p class="govuk-body-m">This website is run by HM Courts & Tribunals Service (HMCTS). We want as many people as possible to be able to use this service. This means you should be able to:</p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li>change colours, contrast levels and fonts</li>
+    <li>zoom in up to 300% without the text spilling off the screen</li>
+    <li>navigate most of the website using just a keyboard</li>
+    <li>navigate most of the website using speech recognition software</li>
+    <li>listen to most of the website using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)</li>
+  </ul>
+  <p class="govuk-body-m">We’ve also made the website text as simple as possible to understand.</p>
+  <p class="govuk-body-m"><a class="govuk-link" href="https://mcmw.abilitynet.org.uk/">AbilityNet</a> has advice on making your device easier to use if you have a disability.</p>
+
+  <h1 class="govuk-heading-xl">How accessible this website is</h1>
```

Comment:
If you look at the main gov uk accessibility website, the main title uses "govuk-heading-xl" but the subsequent headers like this one uses "govuk-heading-m". May be we should follow the same style. At the moment all headers are using "govuk-heading-xl" and it doesn't differentiate between the title and other sub-sections.